### PR TITLE
 Add masterContractHash to relayerConfig

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-ops/src/dev/deployWalletMaster.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-import WalletMaster from '@universal-login/contracts/build/WalletMaster';
+import WalletMaster from '@universal-login/contracts/build/WalletMaster.json';
 import {deployContract} from 'ethereum-waffle';
+import {utils} from 'ethers';
 
 export default async function deployWalletMaster(deployWallet) {
   const {address} = await deployContract(deployWallet, WalletMaster);
   console.log(`WalletMaster address: ${address}`);
-  return address;
+  const masterContractHash = utils.keccak256(`0x${WalletMaster.evm.deployedBytecode.object}`);
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/deployWalletMaster.js]: To consider - extract helpers: 
* `getContractDeplyedBaytecode()`
* `getContractHash()`

File: universal-login-ops/src/dev/startDevelopment.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ const databaseConfig = {
 
 const ensDomains = ['mylogin.eth', 'universal-id.eth', 'popularapp.eth'];
 
-function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddress: string, tokenContractAddress: string, ensAddress: string, ensRegistrars: string[]) {
+function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddress: string, tokenContractAddress: string, ensAddress: string, ensRegistrars: string[], validCodeHashes: ValidCodeHashes) {
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/startDevelopment.ts]: rename `masterWhitelist`

File: universal-login-ops/src/dev/startDevelopment.ts [hunk 3]
```diff
@@ -52,17 +59,27 @@ declare interface startDevelopmentOverrides {
   relayerClass?: RelayerClass;
 }
 
+declare interface ValidCodeHashes {
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/startDevelopment.ts]: Rename `ValidCodeHashes` -> `ContractWhitelist`

File: universal-login-ops/src/dev/startDevelopment.ts [hunk 4]
```diff
@@ -38,10 +38,17 @@ function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddres
       ensAddress,
       chainId: 0
     },
-    ensRegistrars
+    ensRegistrars,
+    validCodeHashes
   };
 }
 
+function getHashDeployedProxyAbicode() {
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/startDevelopment.ts]: Come on! Is that a good name? 'getProxyBytecode()'

File: universal-login-ops/src/dev/startDevelopment.ts [hunk 5]
```diff
@@ -38,10 +38,17 @@ function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddres
...
+  const proxyContractHash = utils.keccak256(`0x${Proxy.evm.deployedBytecode.object}`);
```
Comment (marekkirejczyk) [inline universal-login-ops/src/dev/startDevelopment.ts]: You can now use `getContractHash`

File: universal-login-ops/src/dev/startDevelopment.ts (hunk 2 above)
Comment (xitronix) [reply universal-login-ops/src/dev/startDevelopment.ts]: ✔️

File: universal-login-ops/src/dev/startDevelopment.ts (hunk 4 above)
Comment (xitronix) [reply universal-login-ops/src/dev/startDevelopment.ts]: ✔️

File: universal-login-ops/src/dev/startDevelopment.ts (hunk 3 above)
Comment (xitronix) [reply universal-login-ops/src/dev/startDevelopment.ts]: ✔️

File: universal-login-ops/src/dev/startDevelopment.ts (hunk 5 above)
Comment (xitronix) [reply universal-login-ops/src/dev/startDevelopment.ts]: ✔️

File: universal-login-ops/src/dev/deployWalletMaster.js (hunk 1 above)
Comment (xitronix) [reply universal-login-ops/src/dev/deployWalletMaster.js]: ✔️

**Pull request merged**